src/math/sieve_of_eratosthenes.jl

```diff
@@ -0,0 +1,17 @@
+"""
+Sieve of Eratosthenes is an algorithm for finding all the primes upto a limit `n`.
+
+Reference:
+-https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes
```

Comment:
```suggestion
-https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes

Contributed By:- [Ved Mahajan](https://github.com/Ved-Mahajan)
```

Reply:
I seriously think this is not really what should be the suggestion. I mean, script format can and should depend on the contributor. PR review should on the performance and working of code. And, of course, where it will e in repo and all

Reply:
> PR review should be on the performance and working of code

Yeah, I know, but I want other people to learn from these algorithms. We want them to be fully documented and understandable for other readers, learners, and students. 🙂

I don't really know Julia, so I can't contribute much to reviewing codes. Someone else (maybe you or @Azzaare, or someone else?) has to do the code review.

Reply:
As for the performance you dont have to worry much...this code is same as the code that Stefan wrote in one [stackoverflow post](https://stackoverflow.com/questions/25027495/sieve-of-eratosthenes-speed-comparison-python-vs-julia)